cosmos/config/default.go

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: BUSL-1.1
+//
+// Copyright (C) 2023, Berachain Foundation. All rights reserved.
+// Use of this software is govered by the Business Source License included
+// in the LICENSE file of this repository and at www.mariadb.com/bsl11.
+//
+// ANY USE OF THE LICENSED WORK IN VIOLATION OF THIS LICENSE WILL AUTOMATICALLY
+// TERMINATE YOUR RIGHTS UNDER THIS LICENSE FOR THE CURRENT AND ALL OTHER
+// VERSIONS OF THE LICENSED WORK.
+//
+// THIS LICENSE DOES NOT GRANT YOU ANY RIGHT IN ANY TRADEMARK OR LOGO OF
+// LICENSOR OR ITS AFFILIATES (PROVIDED THAT YOU MAY USE A TRADEMARK OR LOGO OF
+// LICENSOR AS EXPRESSLY REQUIRED BY THIS LICENSE).
+//
+// TO THE EXTENT PERMITTED BY APPLICABLE LAW, THE LICENSED WORK IS PROVIDED ON
+// AN “AS IS” BASIS. LICENSOR HEREBY DISCLAIMS ALL WARRANTIES AND CONDITIONS,
+// EXPRESS OR IMPLIED, INCLUDING (WITHOUT LIMITATION) WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, NON-INFRINGEMENT, AND
+// TITLE.
+
+package config
+
+import (
+	"github.com/berachain/polaris/eth/node"
+	"github.com/berachain/polaris/eth/polar"
+
+	cmtcfg "github.com/cometbft/cometbft/config"
+
+	serverconfig "github.com/cosmos/cosmos-sdk/server/config"
+)
+
+// RecommendedCometBFTConfig returns the recommended CometBFT config
+// for the application.
+func RecommendedCometBFTConfig() *cmtcfg.Config {
+	cfg := cmtcfg.DefaultConfig()
+	cfg.Mempool.Size = 30000
+	cfg.Mempool.CacheSize = 30000
+	cfg.Mempool.Recheck = true
+	cfg.Mempool.Type = "flood"
+
+	cfg.P2P.MaxNumInboundPeers = 10
+	cfg.P2P.MaxNumOutboundPeers = 15
+
+	cfg.TxIndex.Indexer = "null"
+
+	cfg.Instrumentation.Prometheus = true
+	return cfg
+}
+
+// RecommendedServerConfig returns the recommended server config.
+func RecommendedServerConfig() *serverconfig.Config {
+	cfg := serverconfig.DefaultConfig()
+	cfg.MinGasPrices = "0abera"
```

Comment:
It'd probs make sense to increment `MinGasPrices` to avoid potential spam txs from entering the mempl?

Reply:
had a similar thought, does this gate txs from being added to a validator's mempool? is eip-1559 not the general min gas price for txs?

Reply:
we dont actually check this min gas price on the cosmos side. All the gas logic is enforced by geth's txpool using 1559, etc